core/modules/node/node.module

```diff
@@ -3448,6 +3448,12 @@ function node_autoload_info() {
 
     // Views handlers.
     'views_handler_argument_dates_various' => 'views/views_handler_argument_dates_various.inc',
```

Comment:
Thanks @serundeputy! It looks like this class doesn't exist at all. Could we delete this line as well?